Changelog.md

```diff
@@ -0,0 +1,36 @@
+## Release 0.1 (07/01/23)
+
+### Additions
+- Add read, sort and print json data
+- Add tests for functions
+- Add ability to create pie chart
+- Add requirements
+- Add project file
+- Add python import file
+- Add examples to ignore
```

Comment:
I would condense these to a "feature level" overview. What features did we add last time?

Reply:
done